### PR TITLE
Add basic info on getting started with backend development

### PR DESCRIPTION
**Pull request by rroller**
The page was placeholder text. Adding some pointers on how to get started. Nothing groundbreaking, but it's something :)

### CODE REVIEW

File: documentation/developer/backend.md [hunk 1]
```diff
@@ -4,8 +4,6 @@ title: Backend
 description: "Backend development"
 ---
 
-This page should contain:
+Use the [integration blueprint](https://github.com/custom-components/integration_blueprint) to get started - copy it and build out your integration using the blueprint as a base.
```
Comment (ludeeus) [inline documentation/developer/backend.md:7]: These instructions are (or at least should be) for the backend/integration development of HACS itself.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: documentation/developer/backend.md (hunk 1 above)
Comment (rroller) [reply documentation/developer/backend.md:7]: Thanks, I didn't realize that. I kept landing on this page when trying to find details on how to build out an integration. 

Thought's on me rewording this to say something like "This page is mean for docs related to backend development for HACS itself. If you are looking to build an integration, see..."

Or I could move these docs to a new page?

Comment (ludeeus) [reply documentation/developer/backend.md:7]: Rewording is fine.
I'm guessing this is the page you were searching for <https://hacs.xyz/docs/publish/integration> it has resources at the bottom

**Comment by ludeeus** [comment]
Closing as stale

**Pull request closed without merging**